process/coordinator/process_test.go

```diff
@@ -859,17 +867,88 @@ func TestTransactionCoordinator_CreateMbsAndProcessTransactionsFromMe(t *testing
 		return true
 	}
 
+	marshalizer := &mock.MarshalizerMock{}
+	hasher := &mock.HasherMock{}
+	for i := uint32(0); i < nrShards; i++ {
```

Comment:
Renaming `i` to `shardID` would improve the readability of the loop.

Reply:
done